control.lua

```diff
@@ -31,38 +43,51 @@ local spawn_settings = {
 }
 local safe_zone_size = settings.global["safe-zone-size"].value
 local repeat_map = settings.global["repeat-map"].value
+
 local out_of_map_code = "o" -- The terrain to use for everything outside the map
 
+-- log("~world_map: " .. world_map)
+-- log("~map_index: " .. map_index)
+-- log("~detail_factor: " .. detail_factor)
+-- log("~scale: " .. scale)
+-- log("~position: " .. spawn_settings.position)
+-- log("~safe_zone: " .. safe_zone_size)
+
+-- disable new game start stuff
+script.on_init(function()
+    remote.call("freeplay", "set_disable_crashsite", true) -- removes crashsite and cutscene start
+    remote.call("freeplay", "set_skip_intro", true)        -- Skips popup message to press tab to start playing
+  end)
```

Comment:
I disabled it because I found the starting animation annoying and not useful past the first one minute.  It also doesn't fire in a multiplayer map.  I'll remove it, not a difficult change.  And yes, your work does ensure the ship has room to be created.